packages/i/imgui/xmake.lua

```diff
@@ -17,6 +18,10 @@ package("imgui")
                 set_kind("static")
                 add_files("*.cpp")
                 add_headerfiles("imgui.h", "imconfig.h")
+
+                if is_plat("windows", "mingw") then
+                    add_syslinks("Imm32")
+                end
```

Comment:
please move to outside.

```lua
add_versions(...)

if is_plat("windows", "mingw") then
    add_syslinks("Imm32")
end
```

Reply:
I was about to ask if that is the right place. Moved it :)